### DISCUSSION TITLE
Log showing equipped items in d2bot.exe

### DISCUSSION THREAD

**Discussion by jh86** [isid0re/SoloLeveling#384] [Q&A] [answered]
Is there a way to mute all the charms and items the bot says it's equipping over and over? I am currently running 8 characters in SP and the program gets very lagged out because of all the information being logged in there. I checked the standard config and it doesn't show an option to mute these in the log. Does anyone know how this can be achieved? I opened a lot of files exploring but made no progress.

**Comment by isid0re** [comment, marked as answer]
If U go to the itemoverrides.js file in sololeveling/functions folder. Search for logequipped and comment the line out using //

That currently is the only way at the moment. But this sounds like a good configuration option to add.

**Reply by jh86** [reply]
I had to try a few things but this worked so far - under line 600 (line above added for context)

		if (!equipped || this.equipCharm(items[0])) {
						//Misc.logItem("Equipped", me.getItem(-1, -1, items[0].gid));

Thank you!
